dali/test/tensor_list_wrapper.h

```diff
@@ -15,6 +15,8 @@
 #ifndef DALI_TEST_TENSOR_LIST_WRAPPER_H_
 #define DALI_TEST_TENSOR_LIST_WRAPPER_H_
 
+#include <string>
+#include <third_party/googletest/googletest/include/gtest/gtest.h>
```

Comment:
I think that we have google test in our include path, you don't have to spell it out here.

Reply:
Done